rules/aws/credential_access_aws_iam_assume_role_brute_force.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "development"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed attempts to assume an AWS IAM role. IAM roles are used to delegate access to users or
+services. An adversary may attempt to enumerate IAM roles in order to determine whether a role exists or not before
+attempting to assume or hijack the discovered role.
+"""
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "AWS IAM Assume Role Brute Force"
```

Comment:
can you reword this a little to make it easier to understand? a little more sentence-like would help.
I think "AWS IAM Brute Force of the Role Trust Policy"

also, I wonder if we should adopt a naming convention for some of these (can figure this out later). maybe a colon after "AWS" or something?

Reply:
@bm11100 I know that you're familiar with the behavior that this rule will detect. How does @rw-access's rule name suggestion sound to you?

Reply:
I do think his suggestion makes sense. The only thing I’d say is that it doesn’t follow the convention we’ve used thus far for AWS rules. 

We usually name them as service (IAM) -> API (AssumeRole) -> action (brute force). 

Since all the AWS rules are based around various API calls, I think it’s important to have the API in the name. Also, it makes it easy to search. 

If anything, I’d change to “IAM Assume Role Policy Brute Force”. 

I’m all for having a convention though. A colon after AWS would make sense to me too.

Reply:
If we flip the action and API - “AWS IAM Brute Force of Assume Role Policy”

Reply:
Nice. Thanks